### PR TITLE
Reduce cache on Travis

### PR DESCRIPTION
**Pull request by JohnTitor**
Fixes that Travis jobs are terminated

### CODE REVIEW

**Comment by JohnTitor** [comment]
It seems fine, what about? @weiznich

**Comment by JohnTitor** [comment]
Attempt to reduce more following https://levans.fr/rust_travis_cache.html

**Comment by killercup** [comment]
This might also be interesting: https://github.com/rust-analyzer/rust-analyzer/blob/04c2961d0c5493962e948dc8101445cc76f1d460/.travis.yml#L2-L5

**Comment by JohnTitor** [comment]
@killercup Good point! It seems reasonable. If we don't drop Travis right away, we can go for reducing cache for now.

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,15 @@ rust:
...
   - beta
   - nightly
-cache: cargo
+cache:
+  directories:
+    - $HOME/.cargo
+    - $HOME/.rustup
+before_cache:
+  - rm -rf /home/travis/.cargo/registry
+  - find ./target/debug -maxdepth 1 -type f -delete
```
Comment (weiznich) [inline .travis.yml]: We don't cache `target` therefore we don't need to delete there something right?

Comment (JohnTitor) [reply .travis.yml]: Ah, indeed, so we don't need a2d36107e770b07fbf164174727535a00f113871?

Comment (weiznich) [reply .travis.yml]: I think we don't need it.

Comment (JohnTitor) [reply .travis.yml]: Okay, dropped the commit.

**Pull request merged**